### PR TITLE
Integration of azure event hubs in kyma-gardener-azure-integration

### PR DESCRIPTION
**Pull request by sayanh**
**Description**

Changes proposed in this pull request:

- Added integration of Azure Event Hubs in `kyma-gardener-azure-integration` job.

**Related issue(s)**

See https://github.com/kyma-project/kyma/issues/7064

### CODE REVIEW

File: prow/scripts/cluster-integration/helpers/create-azure-event-hubs-secret.sh [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+#
+# Provision A New Azure EventHub Namespace In The Current Azure Subscription
+#
+# Each Azure EventHubs Namespace can contain a maximum of 10 EventHubs (Knative Channels / Kakfa Topics) which equates
+# to unique combinations of a Event Source / Event Type / Event Version.  Because there is an associated cost with
+# empty or unused EventHub Namespaces, we only want to provision the minimum number required.
+#
+# It is expected that prior to running this script the Azure subscription needs to have sufficient permissions
+# to be able to perform the necessary tasks.  Finally the environment should be setup with "az", "kubectl", "jq",
+# and "base64" on their $PATH.
```
Comment (nachtmaar) [inline prow/scripts/cluster-integration/helpers/create-azure-event-hubs-secret.sh]: base64 is not required as far as I can see

File: prow/scripts/cluster-integration/helpers/create-azure-event-hubs-secret.sh [hunk 2]
```diff
@@ -0,0 +1,248 @@
...
+    shout "Executable 'az' Not Found On \$PATH - Exiting"
+    exit 1
+  fi
+
+  if ! [[ -x "$(command -v jq)" ]]; then
+    shout "Executable 'jq' Not Found On \$PATH - Exiting"
+    exit 1
+  fi
+
+  if ! [[ -x "$(command -v base64)" ]]; then
```
Comment (nachtmaar) [inline prow/scripts/cluster-integration/helpers/create-azure-event-hubs-secret.sh]: base64 command is not required as far as I can see.

File: prow/scripts/cluster-integration/helpers/create-azure-event-hubs-secret.sh [hunk 3]
```diff
@@ -0,0 +1,248 @@
...
+  fi
+
+}
+
+
+function azureAuthenticating() {
+	shout "Authenticating to azure"
+	date
+
+	az login --service-principal -u "${AZURE_SUBSCRIPTION_APP_ID}" -p "${AZURE_SUBSCRIPTION_SECRET}" --tenant "${AZURE_SUBSCRIPTION_TENANT}"
```
Comment (nachtmaar) [inline prow/scripts/cluster-integration/helpers/create-azure-event-hubs-secret.sh]: (nit): let's make it multiline.

```suggestion
	az login \
	    --service-principal \
	    -u "${AZURE_SUBSCRIPTION_APP_ID}" \
	    -p "${AZURE_SUBSCRIPTION_SECRET}" \
	    --tenant "${AZURE_SUBSCRIPTION_TENANT}"
```

File: prow/scripts/cluster-integration/helpers/create-azure-event-hubs-secret.sh [hunk 4]
```diff
@@ -0,0 +1,248 @@
...
+}
+
+
+# Create The EventHub Namespace Based On User's Current Azure / K8S Context
+createEventHubNamespace() {
+
+  # Execute The Azure EventHubs Namespace Creation Command & Handle The Results
+  shout "Creating New EventHubs Namespace... (takes several minutes - be patient :)"
+  if [[ $(cmdCreateEventHubNamespace) -eq 0 ]]; then
+    sleep 2
```
Comment (nachtmaar) [inline prow/scripts/cluster-integration/helpers/create-azure-event-hubs-secret.sh]: Why is the sleep required?

File: prow/scripts/cluster-integration/helpers/create-azure-event-hubs-secret.sh [hunk 5]
```diff
@@ -0,0 +1,248 @@
...
+    shout "Failed To Create New EventHub Namespace - Exiting!"
+    exit 1
+  fi
+}
+
+
+# Load The EventHub Namespace's Authorization Key Information Into Global Variables
+loadAuthorizationKey() {
+
+   # Get The New EventHub Namespace's PrimaryConnectionString
```
Comment (nachtmaar) [inline prow/scripts/cluster-integration/helpers/create-azure-event-hubs-secret.sh]: (nit) 

```suggestion
  # Get The New EventHub Namespace's PrimaryConnectionString
```

File: prow/scripts/cluster-integration/helpers/create-azure-event-hubs-secret.sh [hunk 6]
```diff
@@ -0,0 +1,248 @@
...
+  eventHubSecret=$(cmdCreateventHubNamespaceSecret)
+  echo "$eventHubSecret" > "${EVENTHUB_NAMESPACE_NAME}-secret.yaml"
+}
+
+#
+# Main Script Execution
+#
+
+# Authenticating in Azure
+azureAuthenticating
```
Comment (nachtmaar) [inline prow/scripts/cluster-integration/helpers/create-azure-event-hubs-secret.sh]: `verifyPathDependencies` should come before `az` is used in `azureAuthenticating `

File: prow/scripts/cluster-integration/helpers/create-azure-event-hubs-secret.sh [hunk 7]
```diff
@@ -0,0 +1,248 @@
...
+createEventHubNamespace
+
+# Lookup The New Azure EventHub Namespace's Authorization Key
+loadAuthorizationKey
+
+# Create K8S Secret For EventHub Namespace
+createK8SSecretFile
+
+# Exit Successfully
+exit 0
```
Comment (nachtmaar) [inline prow/scripts/cluster-integration/helpers/create-azure-event-hubs-secret.sh]: (nit) This is not required. exit 0 is default.

File: prow/scripts/cluster-integration/helpers/create-azure-event-hubs-secret.sh [hunk 8]
```diff
@@ -0,0 +1,248 @@
...
+# Create The New Azure EventHubs Namespace
+createEventHubNamespace
+
+# Lookup The New Azure EventHub Namespace's Authorization Key
+loadAuthorizationKey
+
+# Create K8S Secret For EventHub Namespace
+createK8SSecretFile
+
+# Exit Successfully
```
Comment (nachtmaar) [inline prow/scripts/cluster-integration/helpers/create-azure-event-hubs-secret.sh]:
```suggestion
```

File: prow/scripts/cluster-integration/helpers/create-azure-event-hubs-secret.sh (hunk 7 above)
Comment (nachtmaar) [inline prow/scripts/cluster-integration/helpers/create-azure-event-hubs-secret.sh]:
```suggestion
```

File: prow/scripts/cluster-integration/helpers/create-azure-event-hubs-secret.sh [hunk 9]
```diff
@@ -0,0 +1,248 @@
...
+
+# Create The New Azure EventHubs Namespace
+createEventHubNamespace
+
+# Lookup The New Azure EventHub Namespace's Authorization Key
+loadAuthorizationKey
+
+# Create K8S Secret For EventHub Namespace
+createK8SSecretFile
+
```
Comment (nachtmaar) [inline prow/scripts/cluster-integration/helpers/create-azure-event-hubs-secret.sh]:
```suggestion
```

File: prow/scripts/cluster-integration/helpers/kyma-cli.sh [hunk 10]
```diff
@@ -8,8 +8,9 @@ install::kyma_cli() {
     pushd "${INSTALL_DIR}/bin"
 
     log::info "- Install kyma CLI ${os} locally to a tempdir..."
-
-    curl -sSLo kyma "https://storage.googleapis.com/kyma-cli-stable/kyma-${os}?alt=media"
+#TODO
+#    curl -sSLo kyma "https://storage.googleapis.com/kyma-cli-stable/kyma-${os}?alt=media"
```
Comment (nachtmaar) [inline prow/scripts/cluster-integration/helpers/kyma-cli.sh]: Kyma cli 1.10.0 is out now. We can revert that hack :)

File: prow/scripts/cluster-integration/helpers/create-azure-event-hubs-secret.sh [hunk 11]
```diff
@@ -0,0 +1,248 @@
...
+    jq -r ".primaryConnectionString"
+}
+
+
+# Create The EventHub Namespace Secret And Return It
+cmdCreateventHubNamespaceSecret() {
+
+  IFS="" read -r -d '' kafkaSecret << EOF
+apiVersion: v1
+stringData:
```
Comment (nachtmaar) [inline prow/scripts/cluster-integration/helpers/create-azure-event-hubs-secret.sh]: stringData is wrong. See [documentation here](https://kyma-project.io/docs/components/event-bus/#details-knative-eventing-mesh-alpha-kafka) for more details.

File: prow/scripts/cluster-integration/helpers/create-azure-event-hubs-secret.sh [hunk 12]
```diff
@@ -0,0 +1,248 @@
...
+  brokers: ${K8S_SECRET_BROKER}
```
Comment (nachtmaar) [inline prow/scripts/cluster-integration/helpers/create-azure-event-hubs-secret.sh]:
```suggestion
  kafka.brokers: ${K8S_SECRET_BROKER}
```

File: prow/scripts/cluster-integration/helpers/create-azure-event-hubs-secret.sh [hunk 13]
```diff
@@ -0,0 +1,248 @@
...
+  namespace: ${EVENTHUB_NAMESPACE_NAME}
```
Comment (nachtmaar) [inline prow/scripts/cluster-integration/helpers/create-azure-event-hubs-secret.sh]:
```suggestion
  kafka.namespace: ${EVENTHUB_NAMESPACE_NAME}
```

File: prow/scripts/cluster-integration/helpers/create-azure-event-hubs-secret.sh [hunk 14]
```diff
@@ -0,0 +1,248 @@
...
+  password: ${K8S_SECRET_PASSWORD}
```
Comment (nachtmaar) [inline prow/scripts/cluster-integration/helpers/create-azure-event-hubs-secret.sh]:
```suggestion
  kafka.password: ${K8S_SECRET_PASSWORD}
```

File: prow/scripts/cluster-integration/helpers/create-azure-event-hubs-secret.sh [hunk 15]
```diff
@@ -0,0 +1,248 @@
...
+  username: ${K8S_SECRET_USERNAME}
```
Comment (nachtmaar) [inline prow/scripts/cluster-integration/helpers/create-azure-event-hubs-secret.sh]:
```suggestion
  kafka.username: ${K8S_SECRET_USERNAME}
  environment.kafkaProvider: azure
  kafka.secretName: knative-kafka
```

File: prow/scripts/cluster-integration/helpers/create-azure-event-hubs-secret.sh [hunk 16]
```diff
@@ -0,0 +1,248 @@
...
+  shout "                              Azure Subscription    :  ${AZURE_SUBSCRIPTION}"
+  shout "                              Azure Resource Group  :  ${RS_GROUP}"
+  shout ""
+  shout "             New EventHub Namespace Name            :  ${EVENTHUB_NAMESPACE_NAME}"
+  shout "             New EventHub Namespace Location        :  ${EVENTHUB_NAMESPACE_LOCATION}"
+  shout "             New EventHub Namespace Throughput Min  :  ${EVENTHUB_NAMESPACE_MIN_THROUGHPUT_UNITS}"
+  shout "             New EventHub Namespace Throughput Max  :  ${EVENTHUB_NAMESPACE_MAX_THROUGHPUT_UNITS}"
+  shout "                       Kubernetes Secret Name       :  ${EVENTHUB_NAMESPACE_NAME}"
+  shout "                       Kubernetes Secret Namespace  :  ${K8S_SECRET_NAMESPACE}"
+  shout "            Kafka Channel Controller Pod Name       :  ${KAFKA_CHANNEL_CONTROLLER_NAME}"
```
Comment (nachtmaar) [inline prow/scripts/cluster-integration/helpers/create-azure-event-hubs-secret.sh]: Variable is not set.

```suggestion
```

File: prow/scripts/cluster-integration/helpers/create-azure-event-hubs-secret.sh [hunk 17]
```diff
@@ -0,0 +1,248 @@
...
+  shout "            Kafka Channel Controller Pod Namespace  :  ${KAFKA_CHANNEL_CONTROLLER_NAMESPACE}"
```
Comment (nachtmaar) [inline prow/scripts/cluster-integration/helpers/create-azure-event-hubs-secret.sh]: Variable is not set.

```suggestion
```

Comment (sayanh) [reply prow/scripts/cluster-integration/helpers/create-azure-event-hubs-secret.sh]: Variable is removed

File: prow/scripts/cluster-integration/helpers/create-azure-event-hubs-secret.sh (hunk 16 above)
Comment (sayanh) [reply prow/scripts/cluster-integration/helpers/create-azure-event-hubs-secret.sh]: Variable is removed

File: prow/scripts/cluster-integration/helpers/create-azure-event-hubs-secret.sh (hunk 15 above)
Comment (sayanh) [reply prow/scripts/cluster-integration/helpers/create-azure-event-hubs-secret.sh]: Fixed

File: prow/scripts/cluster-integration/helpers/create-azure-event-hubs-secret.sh (hunk 14 above)
Comment (sayanh) [reply prow/scripts/cluster-integration/helpers/create-azure-event-hubs-secret.sh]: Fixed

File: prow/scripts/cluster-integration/helpers/create-azure-event-hubs-secret.sh (hunk 13 above)
Comment (sayanh) [reply prow/scripts/cluster-integration/helpers/create-azure-event-hubs-secret.sh]: Fixed

File: prow/scripts/cluster-integration/helpers/create-azure-event-hubs-secret.sh (hunk 12 above)
Comment (sayanh) [reply prow/scripts/cluster-integration/helpers/create-azure-event-hubs-secret.sh]: Fixed

File: prow/scripts/cluster-integration/helpers/create-azure-event-hubs-secret.sh (hunk 11 above)
Comment (sayanh) [reply prow/scripts/cluster-integration/helpers/create-azure-event-hubs-secret.sh]: Fixed

File: prow/scripts/cluster-integration/kyma-gardener-integration.sh [hunk 18]
```diff
@@ -64,6 +76,12 @@ cleanup() {
 
     rm -rf "${TMP_DIR}"
 
+    # Delete the Azure Event Hubs namespace which was created
+    set +e
+    az eventhubs namespace delete -n "${EVENTHUB_NAMESPACE_NAME}" -g "${RS_GROUP}"
+
+    # Delete the Azure Event Hubs namespace which was created
```
[the remark this answers is not in the document]
Comment (sayanh) [reply prow/scripts/cluster-integration/kyma-gardener-integration.sh]: Fixed

File: prow/scripts/cluster-integration/kyma-gardener-integration.sh [hunk 19]
```diff
@@ -107,16 +125,23 @@ kyma provision gardener \
...
 curl -L --silent --fail --show-error "https://raw.githubusercontent.com/kyma-project/kyma/master/installation/resources/installer-config-production.yaml.tpl" \
     --output installer-config-production.yaml.tpl
 
+curl -L --silent --fail --show-error "https://raw.githubusercontent.com/kyma-project/kyma/master/installation/resources/installer-cr-gardener-azure.yaml.tpl" \
+    --output installer-cr-gardener-azure.yaml.tpl
+
 (
 set -x
-yes | kyma install --non-interactive --source latest -o installer-config-production.yaml.tpl --timeout 90m
+yes | kyma install --non-interactive --source latest -o installer-cr-gardener-azure.yaml.tpl -o installer-config-production.yaml.tpl -o "${EVENTHUB_NAMESPACE_NAME}-secret.yaml" --timeout 90m
```
[the remark this answers is not in the document]
Comment (sayanh) [reply prow/scripts/cluster-integration/kyma-gardener-integration.sh]: Done

File: prow/scripts/cluster-integration/kyma-gardener-integration.sh [hunk 20]
```diff
@@ -64,6 +76,12 @@ cleanup() {
 
     rm -rf "${TMP_DIR}"
 
+    # Delete the Azure Event Hubs namespace which was created
+    set +e
```
[the remark this answers is not in the document]
Comment (sayanh) [reply prow/scripts/cluster-integration/kyma-gardener-integration.sh]: Rearranged the lines hence this is not needed anymore.

File: prow/scripts/cluster-integration/kyma-gardener-integration.sh [hunk 21]
```diff
@@ -29,7 +36,12 @@ if [ "${discoverUnsetVar}" = true ] ; then
     exit 1
 fi
 
+
+#AZURE_SUBSCRIPTION=?
+
+
 #Exported variables
+export RS_GROUP EVENTHUB_NAMESPACE_NAME REGION AZURE_SUBSCRIPTION_ID AZURE_SUBSCRIPTION_APP_ID AZURE_SUBSCRIPTION_SECRET AZURE_SUBSCRIPTION_TENANT
```
[the remark this answers is not in the document]
Comment (sayanh) [reply prow/scripts/cluster-integration/kyma-gardener-integration.sh]: Changed

File: prow/scripts/cluster-integration/kyma-gardener-integration.sh [hunk 22]
```diff
@@ -29,7 +36,12 @@ if [ "${discoverUnsetVar}" = true ] ; then
     exit 1
 fi
 
+
+#AZURE_SUBSCRIPTION=?
```
[the remark this answers is not in the document]
Comment (sayanh) [reply prow/scripts/cluster-integration/kyma-gardener-integration.sh]: Removed

File: prow/scripts/cluster-integration/kyma-gardener-integration.sh [hunk 23]
```diff
@@ -12,14 +12,21 @@
...
+# - EVENTHUB_NAMESPACE_NAME
 #
 #Permissions: In order to run this script you need to use an AKS service account with the contributor role
 
 set -o errexit
 
 discoverUnsetVar=false
 
-for var in KYMA_PROJECT_DIR GARDENER_REGION GARDENER_KYMA_PROW_KUBECONFIG GARDENER_KYMA_PROW_PROJECT_NAME GARDENER_KYMA_PROW_PROVIDER_SECRET_NAME; do
+for var in KYMA_PROJECT_DIR GARDENER_REGION GARDENER_KYMA_PROW_KUBECONFIG GARDENER_KYMA_PROW_PROJECT_NAME GARDENER_KYMA_PROW_PROVIDER_SECRET_NAME RS_GROUP EVENTHUB_NAMESPACE_NAME REGION AZURE_SUBSCRIPTION_ID AZURE_SUBSCRIPTION_APP_ID AZURE_SUBSCRIPTION_SECRET AZURE_SUBSCRIPTION_TENANT; do
```
[the remark this answers is not in the document]
Comment (sayanh) [reply prow/scripts/cluster-integration/kyma-gardener-integration.sh]: Changed

File: prow/scripts/cluster-integration/helpers/create-azure-event-hubs-secret.sh [hunk 24]
```diff
@@ -0,0 +1,248 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o pipefail  # Fail a pipe if any sub-command fails.
+
+#!/bin/bash
```
[the remark this answers is not in the document]
Comment (sayanh) [reply prow/scripts/cluster-integration/helpers/create-azure-event-hubs-secret.sh]: Not needed hence removed.

File: prow/scripts/cluster-integration/helpers/create-azure-event-hubs-secret.sh [hunk 25]
```diff
@@ -0,0 +1,248 @@
...
+# It is expected that prior to running this script the Azure subscription needs to have sufficient permissions
+# to be able to perform the necessary tasks.  Finally the environment should be setup with "az", "kubectl", "jq",
+# and "base64" on their $PATH.
+#
+########################################################################################################################
+
+set -e
+
+# Uncomment To Debug (Verbose!)
+#set -x
```
[the remark this answers is not in the document]
Comment (sayanh) [reply prow/scripts/cluster-integration/helpers/create-azure-event-hubs-secret.sh]: Removed

File: prow/scripts/cluster-integration/helpers/create-azure-event-hubs-secret.sh [hunk 26]
```diff
@@ -0,0 +1,248 @@
...
+#
+# It is expected that prior to running this script the Azure subscription needs to have sufficient permissions
+# to be able to perform the necessary tasks.  Finally the environment should be setup with "az", "kubectl", "jq",
+# and "base64" on their $PATH.
+#
+########################################################################################################################
+
+set -e
+
+# Uncomment To Debug (Verbose!)
```
[the remark this answers is not in the document]
Comment (sayanh) [reply prow/scripts/cluster-integration/helpers/create-azure-event-hubs-secret.sh]: Removed

File: prow/scripts/cluster-integration/helpers/create-azure-event-hubs-secret.sh [hunk 27]
```diff
@@ -0,0 +1,248 @@
...
+# to unique combinations of a Event Source / Event Type / Event Version.  Because there is an associated cost with
+# empty or unused EventHub Namespaces, we only want to provision the minimum number required.
+#
+# It is expected that prior to running this script the Azure subscription needs to have sufficient permissions
+# to be able to perform the necessary tasks.  Finally the environment should be setup with "az", "kubectl", "jq",
+# and "base64" on their $PATH.
+#
+########################################################################################################################
+
+set -e
```
[the remark this answers is not in the document]
Comment (sayanh) [reply prow/scripts/cluster-integration/helpers/create-azure-event-hubs-secret.sh]: Removed

File: prow/scripts/cluster-integration/helpers/create-azure-event-hubs-secret.sh [hunk 28]
```diff
@@ -0,0 +1,248 @@
...
+
+#
+# Utility Functions To Make The Actual Cmd Line Calls
+#
+
+createGroup() {
+	shout "Create Azure group"
+	date
+
+	az group create --name "${RS_GROUP}" --location "${REGION}"
```
[the remark this answers is not in the document]
Comment (sayanh) [reply prow/scripts/cluster-integration/helpers/create-azure-event-hubs-secret.sh]: Changed

File: prow/scripts/cluster-integration/helpers/create-azure-event-hubs-secret.sh (hunk 1 above)
Comment (sayanh) [reply prow/scripts/cluster-integration/helpers/create-azure-event-hubs-secret.sh]: Removed

File: prow/scripts/cluster-integration/helpers/create-azure-event-hubs-secret.sh [hunk 29]
```diff
@@ -0,0 +1,248 @@
...
+K8S_SECRET_USERNAME="\$ConnectionString"
+K8S_SECRET_BROKER=""
+K8S_SECRET_PASSWORD=""
+
+#
+# Utility Functions To Make The Actual Cmd Line Calls
+#
+
+createGroup() {
+	shout "Create Azure group"
```
[the remark this answers is not in the document]
Comment (sayanh) [reply prow/scripts/cluster-integration/helpers/create-azure-event-hubs-secret.sh]: Changed tabs to spaces and made it consistent throughout the file

File: prow/scripts/cluster-integration/helpers/create-azure-event-hubs-secret.sh (hunk 3 above)
Comment (sayanh) [reply prow/scripts/cluster-integration/helpers/create-azure-event-hubs-secret.sh]: Changed

File: prow/scripts/cluster-integration/helpers/create-azure-event-hubs-secret.sh (hunk 2 above)
Comment (sayanh) [reply prow/scripts/cluster-integration/helpers/create-azure-event-hubs-secret.sh]: Removed

File: prow/scripts/cluster-integration/helpers/create-azure-event-hubs-secret.sh (hunk 4 above)
Comment (sayanh) [reply prow/scripts/cluster-integration/helpers/create-azure-event-hubs-secret.sh]: Removed

File: prow/scripts/cluster-integration/helpers/create-azure-event-hubs-secret.sh (hunk 5 above)
Comment (sayanh) [reply prow/scripts/cluster-integration/helpers/create-azure-event-hubs-secret.sh]: Fixed the indentation

File: prow/scripts/cluster-integration/helpers/create-azure-event-hubs-secret.sh (hunk 6 above)
Comment (sayanh) [reply prow/scripts/cluster-integration/helpers/create-azure-event-hubs-secret.sh]: Made this step the first one to get executed

File: prow/scripts/cluster-integration/helpers/create-azure-event-hubs-secret.sh (hunk 7 above)
Comment (sayanh) [reply prow/scripts/cluster-integration/helpers/create-azure-event-hubs-secret.sh]: Removed

File: prow/scripts/cluster-integration/helpers/create-azure-event-hubs-secret.sh (hunk 8 above)
Comment (sayanh) [reply prow/scripts/cluster-integration/helpers/create-azure-event-hubs-secret.sh]: Removed

File: prow/scripts/cluster-integration/helpers/create-azure-event-hubs-secret.sh (hunk 7 above)
Comment (sayanh) [reply prow/scripts/cluster-integration/helpers/create-azure-event-hubs-secret.sh]: Removed

File: prow/scripts/cluster-integration/helpers/create-azure-event-hubs-secret.sh (hunk 9 above)
Comment (sayanh) [reply prow/scripts/cluster-integration/helpers/create-azure-event-hubs-secret.sh]: Removed

File: prow/scripts/cluster-integration/helpers/kyma-cli.sh (hunk 10 above)
Comment (sayanh) [reply prow/scripts/cluster-integration/helpers/kyma-cli.sh]: Reverted

File: prow/scripts/cluster-integration/kyma-gardener-integration.sh [hunk 30]
```diff
@@ -108,20 +141,40 @@ kyma provision gardener \
...
-        --nodes 1 \
+        --nodes 4 \
         --kube-version=${CLUSTER_VERSION}
 )
 
 shout "Installing Kyma"
 date
 
+shout "Downloading Kyma installer CR"
+curl -L --silent --fail --show-error "https://raw.githubusercontent.com/sayanh/kyma/integration-azure-event-hubs/installation/resources/installer-cr-gardener-azure.yaml.tpl" \
```
Comment (sayanh) [inline prow/scripts/cluster-integration/kyma-gardener-integration.sh]: Change it to master once https://github.com/kyma-project/kyma/pull/7205/files gets merged

File: prow/scripts/cluster-integration/kyma-gardener-integration.sh [hunk 31]
```diff
@@ -108,20 +141,40 @@ kyma provision gardener \
...
+    --output installer-cr-gardener-azure.yaml.tpl
+
 echo "Downlading production profile"
 curl -L --silent --fail --show-error "https://raw.githubusercontent.com/kyma-project/kyma/master/installation/resources/installer-config-production.yaml.tpl" \
     --output installer-config-production.yaml.tpl
 
+shout "Downloading Azure EventHubs config"
+curl -L --silent --fail --show-error "https://raw.githubusercontent.com/sayanh/kyma/integration-azure-event-hubs/installation/resources/installer-config-azure-eventhubs.yaml.tpl" \
```
Comment (nachtmaar) [inline prow/scripts/cluster-integration/kyma-gardener-integration.sh]: > Change it to master once https://github.com/kyma-project/kyma/pull/7205/files gets merged

Same here :)

File: prow/scripts/cluster-integration/kyma-gardener-integration.sh (hunk 30 above)
Comment (sayanh) [reply prow/scripts/cluster-integration/kyma-gardener-integration.sh]: Done

File: prow/scripts/cluster-integration/kyma-gardener-integration.sh (hunk 31 above)
Comment (sayanh) [reply prow/scripts/cluster-integration/kyma-gardener-integration.sh]: Done

**Pull request merged**